src/Pages/FilterClassByType.jsx

```diff
@@ -0,0 +1,79 @@
+import { useState } from "react";
+import FormGroup from "@mui/material/FormGroup";
+import FormControlLabel from "@mui/material/FormControlLabel";
+import Checkbox from "@mui/material/Checkbox";
+import { Button, stepConnectorClasses } from "@mui/material";
+
+export default function FilterClassByType(props) {
+  const { classes, setClasses } = props;
+
+  const [filter, setFilter] = useState([]);
+  
+  const handleChange = (event, oneClass) => {
+      
+      if(event.target.checked !== null) {
```

Comment:
Have you tried console logging `event.target.checked`?

Reply:
i did, changed that to `false` wether now am getting to keep the sam unique value even if  I unselect (works for only one unselect/ if select the CB again after it's been unselected will get add again the same obj to the array)